src/CoreBundle/Session/ChameleonSessionManager.php

```diff
@@ -175,4 +178,35 @@ public function isSessionStarting()
     {
         return $this->isSessionStarting;
     }
+
+    /**
+     * @return string
+     */
+    private function getCookieName()
+    {
+        if ($this->requestInfoService->isBackendMode()) {
+            return self::SESSION_BASE_NAME . 'CMS';
+        }
+
+        if ($this->requestInfoService->isCmsTemplateEngineEditMode()) {
+            $this->logger->notice('Using backend session because we are in template engine edit mode');
```

Comment:
Why (notice) logging)? This is normal system function and nothing noteable?

you could join the 3 cases then which makes it more clear that the behave the same.

Reply:
This was the result of a discussion we had about this topic. Because these methods are triggered globally by the existence of certain GET-Parameters, collisions with parameters in custom logic is possible. This would lead to very weird session issues. To make debugging them easier, a notice is being logged for these instances.

Reply:
Hm, not sure I would expect anything helping me debug as a notice in a log...

Reply:
It doesn't hurt, does it? And potentially it saves us a bit of deuabugging in the future